tls/s2n_early_data_io.c

```diff
@@ -38,6 +38,15 @@ int s2n_end_of_early_data_recv(struct s2n_connection *conn)
     return S2N_SUCCESS;
 }
 
+bool s2n_is_rejected_early_data(struct s2n_connection *conn)
```

Comment:
why not s2n_is_early_data_rejected()? Also, this function returns true if the early data was rejected? Something about this comment really confuses me:
/* Early data can only be accepted before the end of the handshake. */
This function is figuring out if early data was rejected or not, maybe don't throw in the word accepted 😄

Reply:
For the naming: we could follow the other early data naming and use `s2n_early_data_is_rejected` instead.